src/Spaces/pointspace.jl

```diff
@@ -0,0 +1,17 @@
+abstract type AbstractPointSpace <: AbstractSpace end
+
+local_geometry_data(space::AbstractPointSpace) = space.local_geometry
+
+
+struct PointSpace{LG} <: AbstractPointSpace
+    local_geometry::LG
+end
+
+function PointSpace(coord::Geometry.Abstract1DPoint{FT}) where {FT}
+    CoordType = typeof(coord)
+    AIdx = Geometry.coordinate_axis(CoordType)
+    LG = Geometry.LocalGeometry{AIdx, CoordType, FT, SMatrix{1, 1, FT, 1}}
+    local_geometry = DataLayouts.DataF{LG}([Geometry.component(coord, 1) 1.0 1.0 1.0 1.0])
```

Comment:
Does this seem like a fair way to initialize?

Reply:
I would suggest defining a constuctor:
```
PointSpace(lg::LocalGeometry)
```
and then defining other constructors in terms of that.

Reply:
Makes sense!

Reply:
Is there any reason not to simply make that kind of type assertion for the default constructor? e.g.
```
struct PointSpace{LG <: Geometry.LocalGeometry} <: AbstractPointSpace
    local_geometry::LG
end
```

Reply:
e.g. something like
```julia
function PointSpace(local_geometry::LocalGeometry)
    local_geometry_data = DataF{typeof(local_geometry(}()
    local_geometry_data[] = local_geometry
    PointSpace(local_geometry_data)
end
```